### PR TITLE
Docs: Add api keys roles, actions and scopes documentation

### PR DESCRIPTION
**Pull request by kalleep**
**What this PR does / why we need it**:
I noticed that api keys permissions where not documented for access control.
* Add fixed roles
* Add all available permissions
* Fix scopes for annotations

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by chri2547 — APPROVED** [review]
LGTM! Ship it!

File: docs/sources/enterprise/access-control/custom-role-actions-scopes.md [hunk 1]
```diff
@@ -38,10 +38,13 @@ The following list contains role-based access control actions.
...
+| `annotations:read`                   | `annotations:*`<br>`annotations:type:*`                                                 | Read annotations and annotation tags.                                                                                                                                                            |
+| `annotations:write`                  | `annotations:*`<br>`annotations:type:*`                                                 | Update annotations.                                                                                                                                                                              |
+| `apikeys:create`                     | n/a                                                                                     | Create api keys                                                                                                                                                                                  |
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/access-control/custom-role-actions-scopes.md:45]:
```suggestion
| `apikeys:create`                     | n/a                                                                                     | Create API keys.                                                                                                                                                                                  |
```

File: docs/sources/enterprise/access-control/custom-role-actions-scopes.md [hunk 2]
```diff
@@ -38,10 +38,13 @@ The following list contains role-based access control actions.
...
+| `apikeys:read`                       | `apikeys:*`<br>`apikeys:id:*`                                                           | Read api keys                                                                                                                                                                                    |
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/access-control/custom-role-actions-scopes.md:46]:
```suggestion
| `apikeys:read`                       | `apikeys:*`<br>`apikeys:id:*`                                                           | Read API keys.                                                                                                                                                                                    |
```

File: docs/sources/enterprise/access-control/custom-role-actions-scopes.md [hunk 3]
```diff
@@ -38,10 +38,13 @@ The following list contains role-based access control actions.
...
+| `apikeys:delete`                     | `apikeys:*`<br>`apikeys:id:*`                                                           | Delete api keys                                                                                                                                                                                  |
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/access-control/custom-role-actions-scopes.md:47]:
```suggestion
| `apikeys:delete`                     | `apikeys:*`<br>`apikeys:id:*`                                                           | Delete API keys.                                                                                                                                                                                  |
```

File: docs/sources/enterprise/access-control/custom-role-actions-scopes.md [hunk 4]
```diff
@@ -137,6 +140,7 @@ The following list contains role-based access control scopes.
 | Scopes                                    | Descriptions                                                                                                                                                                                                                                       |
 | ----------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `annotations:*`<br>`annotations:type:*`   | Restrict an action to a set of annotations. For example, `annotations:*` matches any annotation, `annotations:type:dashboard` matches annotations associated with dashboards and `annotations:type:organization` matches organization annotations. |
+| `apikeys:*`<br>`apikeys:id:*`             | Restrict an action to a set of api keys. For example, `apikeys:*` matches any api key, `apikey:id:1` matches the api key whose id is `1`                                                                                                           |
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/access-control/custom-role-actions-scopes.md:143]:
```suggestion
| `apikeys:*`<br>`apikeys:id:*`             | Restrict an action to a set of API keys. For example, `apikeys:*` matches any API key, `apikey:id:1` matches the API key whose id is `1`.                                                                                                           |
```

**Review by achatterjee-grafana — DISMISSED** [review]
Added some comments. Change all instances of api to API, unless it is part of the code. Thanks!

File: docs/sources/enterprise/access-control/rbac-fixed-basic-role-definitions.md [hunk 5]
```diff
@@ -32,8 +32,10 @@ The following tables list permissions associated with basic and fixed roles.
...
+| `fixed:apikeys:reader`                 | `apikeys:read` for scope `apikeys:*`                                                                                                                                                                                                                                     | Read all api keys                                                                                                                                                                                                                                                                     |
+| `fixed:apikeys:writer`                 | All permissions from `fixed:apikeys:reader` and <br> `apikeys:create` <br> `apikeys:delete` for scope `apikeys:*`                                                                                                                                                        | Read, create, delete all api keys                                                                                                                                                                                                                                                     |
```
Comment (gamab) [inline docs/sources/enterprise/access-control/rbac-fixed-basic-role-definitions.md:38]: Extreme nitpicking :sweat_smile: 
```suggestion
| `fixed:apikeys:reader`                 | `apikeys:read` for scope `apikeys:*`                                                                                                                                                                                                                                     | Read all api keys.                                                                                                                                                                                                                                                                    |
| `fixed:apikeys:writer`                 | All permissions from `fixed:apikeys:reader` and <br> `apikeys:create` <br> `apikeys:delete` for scope `apikeys:*`                                                                                                                                                        | Read, create, delete all api keys.                                                                                                                                                                                                                                                    |
```

**Review by gamab — APPROVED** [review]

**Comment by kalleep** [comment]
Hey @achatterjee-grafana can you review again after new changes and unblock if its good?

**Review by eleijonmarck — APPROVED** [review]

**Pull request merged**